app/models/audio_version.rb

```diff
@@ -64,6 +64,13 @@ def set_status
       return
     end
 
+    pos = 0
+    if audio_files.detect { |af| af.position != (pos += 1) }
+      self.status = INVALID
+      self.status_message = "Audio file missing for position #{pos}"
```

Comment:
It doesn't actually matter.  But I might prefer this error be _after_ the audio-version-template errors in priority.  Meaning you'd get the current "Audio version has 2 files but must have 3 files" error before this one.

Reply:
thanks, I think you're right - I'll move it.